src/nvim/os/fs.c

```diff
@@ -148,11 +148,7 @@ int os_nodetype(const char *name)
     case UV_TCP:         // unix only
     case UV_UNKNOWN_HANDLE:
     default:
-#ifdef WIN32
-      nodetype = NODE_OTHER;
```

Comment:
I think you're right on all accounts. UV_UNKNOWN_HANDLE is arguably the equivalent of INVALID_HANDLE_VALUE--though we are potentially missing some granularity here, but as you said, it is unlikely that NODE_OTHER is ever reached in the Vim function.

So this line should change to NODE_NORMAL (for Windows only).